app/src/main/java/eu/kanade/tachiyomi/ui/main/MainActivity.kt

```diff
@@ -56,8 +57,11 @@ import kotlinx.android.synthetic.main.main_activity.tabs
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.flow.launchIn
 import timber.log.Timber
+import android.content.pm.PackageManager
+import android.content.ComponentName
+import android.content.res.Configuration
 
-class MainActivity : BaseActivity<MainActivityBinding>() {
+open class MainActivity : BaseActivity<MainActivityBinding>() {
```

Comment:
Is this needed?

Reply:
I see now, there is a red activity at the bottom

Reply:
Red is an illusion